### DISCUSSION TITLE
Support for App Tracking Transparency

### DISCUSSION THREAD

**Discussion by pakgva** [oddbit/flutter_facebook_app_events#118] [Q&A]
Starting with iOS 14.5, apps on the App Store need to receive the user’s permission through the AppTrackingTransparency framework before collecting data used to track them. This requirement protects the privacy of App Store users.

**Comment by kobe-fabiohideki** [comment]
> Starting with iOS 14.5, apps on the App Store need to receive the user’s permission through the AppTrackingTransparency framework before collecting data used to track them. This requirement protects the privacy of App Store users.

have you tried to use this package: https://pub.dev/packages/app_tracking_transparency ?
